# Non-Graded Challenge 5

`M Naufal Indriatmoko`  -- *FTDS batch 10*

In [1]:
import numpy as np

Apakah matrix [[1],[3]] adalah eigenvector dari [[1, -1], [6, 4]]? Jika iya, berapakah eigenvalue-nya?

Jawaban:

In [8]:
b = np.array([[1], [3]])
b

array([[1],
       [3]])

In [2]:
a = np.array([[1, -1], [6,4]])
a

array([[ 1, -1],
       [ 6,  4]])

In [4]:
values, vector = np.linalg.eig(a)

In [5]:
print(values)

[2.5+1.93649167j 2.5-1.93649167j]


In [10]:
print(abs(values))

[3.16227766 3.16227766]


In [6]:
print(vector)

[[-0.23145502+0.29880715j -0.23145502-0.29880715j]
 [ 0.9258201 +0.j          0.9258201 -0.j        ]]


In [9]:
print(abs(vector))

[[0.37796447 0.37796447]
 [0.9258201  0.9258201 ]]


terlihat bahwa matrix [[1], [3]] bukan merupakan eigen vector dari matrix [[1, -1], [6, 4]].

eigen vectornya adalah [[0.3779, 0.3779], [0.9258, 0.9258]]

dan eigen valuenya 3.1622